systers_portal/templates/community/page.html

```diff
@@ -0,0 +1,15 @@
+{% extends "community/base.html" %}
```

Comment:
Please fix indentation, add empty lines for readability where necessary, add an empty line on the last line.